docs/manila-csi-plugin/using-manila-csi-plugin.md

```diff
@@ -50,6 +50,7 @@ Parameter | Required | Description
 `type` | _yes_ | Manila [share type](https://wiki.openstack.org/wiki/Manila/Concepts#share_type)
 `shareNetworkID` | _no_ | Manila [share network ID](https://wiki.openstack.org/wiki/Manila/Concepts#share_network)
 `availability` | _no_ | Manila availability zone of the provisioned share. If none is provided, the default Manila zone will be used. Note that this parameter is opaque to the CO and does not influence placement of workloads that will consume this share, meaning they may be scheduled onto any node of the cluster. If the specified Manila AZ is not equally accessible from all compute nodes of the cluster, use [Topology-aware dynamic provisioning](#topology-aware-dynamic-provisioning).
+`autoTopology` | _no_ | When set to "true" and the `availability` parameter is empty, the Manila CSI controller will map the Manila availability zone to a corresponding Nova availability zone.
```

Comment:
`the Manila CSI controller will map the Manila availability zone to a corresponding Nova availability zone.` 

is it to `a corresponding` or it's fixed since it's the target node (nova compute node)'s AZ ,so maybe state the info here? maybe point here to the new section you created..

Reply:
I can rephrase this to a "target node".

Reply:
addressed